src/screens/updatePlanItem/PlanItemTimer.tsx

```diff
@@ -2,67 +2,62 @@ import React from 'react';
 import {
   StyleSheet,
   Text,
-  TimePickerAndroid,
   TouchableOpacity,
 } from 'react-native';
 
-import { Icon, StyledText } from 'components';
+import { Icon } from 'components';
 import { PlanItem } from 'models';
+import { palette } from 'styles';
+import { TimePicker } from './TimePicker';
 
 interface Props {
   planItem: PlanItem;
 }
 
 interface State {
-  itemTime: number;
-  itemTimeText: string;
+  visible: boolean;
 }
 
 export class PlanItemTimer extends React.PureComponent<Props, State> {
+  minutesPicker: any;
```

Comment:
The code should not have any `any` entries. It would be nice to see typing here.